src/oidc/oidc.ts

```diff
@@ -287,14 +287,15 @@ export const OAuth2Logout = async (options: OAuth2LogoutOptions) => {
     const oidcEndpoints = localStorage.getItem('config.oidc_endpoints') || '{}';
 
     let logoutUrl = getOAuthLogoutUrl() || JSON.parse(oidcEndpoints).end_session_endpoint;
-    const userManager = await createUserManager({
-        redirectCallbackUri: options.redirectCallbackUri,
-        postLogoutRedirectUri: options.postLogoutRedirectUri,
-    });
-    const userState = await userManager.getUser();
-    if (userState?.id_token) {
-        logoutUrl += `?id_token_hint=${userState.id_token}&post_logout_redirect_uri${options.postLogoutRedirectUri}`;
-    }
+    // NOTE: Comment this out once front channel is implemented
+    // const userManager = await createUserManager({
+    //     redirectCallbackUri: options.redirectCallbackUri,
+    //     postLogoutRedirectUri: options.postLogoutRedirectUri,
+    // });
+    // const userState = await userManager.getUser();
+    // if (userState?.id_token) {
+    //     logoutUrl += `?id_token_hint=${userState.id_token}&post_logout_redirect_uri=${options.postLogoutRedirectUri}`;
```

Comment:
We can't use this for now since this is related to the front channels feature, disabling it until we have merged the front channels page